analyzer/src/main/kotlin/managers/Sbt.kt

```diff
@@ -63,11 +63,14 @@ class Sbt(
         // https://www.scala-sbt.org/1.x/docs/Command-Line-Reference.html#Command+Line+Options.
         private val CI_MODE = "-Dsbt.ci=true".addQuotesOnWindows()
 
+        // Disable console colors explicitly
```

Comment:
Maybe append some like "... to be on the safe side." to also explain here why this is done in addition to setting `CI_MODE`. Note also the dot at the end of the comment sentence.

Reply:
Commit message nit: "SBT" should be "Sbt" to exactly reflect the class name.

Reply:
Done